### PR TITLE
Migrate hardcoded images to Maven properties

### PR DESCRIPTION
**Pull request by gtroitsk**
### Summary

Migrates all hardcoded images with versions into POM

It is not possible to get rid of hardcoded images in JaegerContainer, KeycloakContainer, AmqContainer, because FW system properties  will be ignored while using them in TS

Please check the relevant options

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Dependency update
- [x] Refactoring
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Example scenarios has been updated / added
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by michalvavrik** [comment]
> Migrates all hardcoded images with versions into BOM

POM

File: examples/database-postgresql/src/test/java/io/quarkus/qe/database/postgresql/OpenShiftPostgresqlIT.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class OpenShiftPostgresqlIT extends AbstractSqlDatabaseIT {
 
     private static final int POSTGRESQL_PORT = 5432;
 
-    @Container(image = "quay.io/quarkusqeteam/postgres:14.0", port = POSTGRESQL_PORT, expectedLog = "is ready")
+    @Container(image = "${postgresql.quarkusqe.image}", port = POSTGRESQL_PORT, expectedLog = "is ready")
```
Comment (michalvavrik) [inline examples/database-postgresql/src/test/java/io/quarkus/qe/database/postgresql/OpenShiftPostgresqlIT.java:14]: Did you try to get rid of this image? I don't see explanation here https://github.com/quarkus-qe/quarkus-test-framework/pull/456 but no idea why we should use our own 2 years old image. Please try.

Comment (gtroitsk) [reply examples/database-postgresql/src/test/java/io/quarkus/qe/database/postgresql/OpenShiftPostgresqlIT.java:14]: Interesting, I look at it again and yes, we can get rid of our QE image. It requires adding `withProperty("PGDATA", "/tmp/psql")` to PostgresqlService. You can see explanation from https://github.com/quarkus-qe/quarkus-test-suite/pull/634

Comment (michalvavrik) [reply examples/database-postgresql/src/test/java/io/quarkus/qe/database/postgresql/OpenShiftPostgresqlIT.java:14]: alright, thank you, go for it

**Comment by gtroitsk** [comment]
run tests

**Review by michalvavrik — APPROVED** [review]
LGTM

**Pull request merged**